ansible/roles/yarn/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+---
+# Node
+- name: Register NodeJS
+  shell: 'curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -'
+  changed_when: false
+
+- name: Remove cmdtest
+  apt:
+    name: cmdtest
+    state: absent
+
+- name: Install Yarn APT key
+  apt_key:
+    data: "{{ lookup('file', 'pubkey.gpg') }}"
+    state: present
+
+- name: Add yarn repository
+  apt_repository:
+    repo: 'deb https://dl.yarnpkg.com/debian/ stable main'
+    update_cache: yes
```

Comment:
It would be worth setting a cache time here like in the other `apt_repository` calls.

Reply:
There's no cache time listed on http://docs.ansible.com/ansible/latest/apt_repository_module.html, and I don't see any examples of this in the other roles for Jessie backports and Dotdeb?

Reply:
You are totally right, ignore this one.